server/src/test/java/org/elasticsearch/index/mapper/LongIndexingDocTests.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.index.mapper;
+
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.index.IndexResponse;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.common.xcontent.XContentFactory;
+import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.rest.RestStatus;
+import org.elasticsearch.test.ESSingleNodeTestCase;
+
+public class LongIndexingDocTests extends ESSingleNodeTestCase {
```

Comment:
nit: Since this is testing NumberFieldMapper behaviour, would it make sense to put this test case in `NumberFieldMapperTests` instead? We wouldn't need a new isolated test class with all the setup/teardown overhead for it in that case.

Reply:
I can only reproduce if the test is of the type `ESSingleNodeTestCase` which is not the case for `NumberFieldMapperTests`

Reply:
Strange, from my IDE it looked like it extends "ESSingleNodeTestCase". Let me try again...

Reply:
Works for me :scratcheshead: I moved your test over and only had to change the index name since the "test" index seems to be in use elsewhere already. Failed when removing the new exception type from the catch statement, passes otherwise.

Reply:
You are totally right, I have moved the test so we don't add a new whole class. Thanks @cbuescher!